shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -162,6 +166,18 @@ - (instancetype)initWithName:(NSString*)labelPrefix
   return self;
 }
 
+- (void)recreatePlatformViewController {
+  _renderingApi = flutter::GetRenderingAPIForProcess(FlutterView.forceSoftwareRendering);
+  _surfaceFactory = flutter::IOSSurfaceFactory::Create(_renderingApi);
+  auto pvc = new flutter::FlutterPlatformViewsController(_surfaceFactory);
```

Comment:
nit: You aren't suppose to use abbreviations in variable names

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Good catch. Removed this now.